amqp/src/main/java/io/github/tcdl/msb/adapters/amqp/AmqpAdapterFactory.java

```diff
@@ -96,6 +94,19 @@ protected ConnectionFactory createConnectionFactory(AmqpBrokerConfig adapterConf
         if(virtualHost.isPresent()) {
             connectionFactory.setVirtualHost(virtualHost.get());
         }
+
+        try {
+            if (adapterConfig.useSSL()) {
+                LOG.info("Configuring adapter to use SSL...");
+                connectionFactory.useSslProtocol();
```

Comment:
I think we are. 

This change was intended as a quick fix the the blocker Dragons have. Their server uses TLSv1. Also this is the default value in amqp client which suggests that it should probably be good enough for most cases.

I agree that for full SSL support we should allow developers to specify the encryption they would like to use. But that should be a part of separate story for implementation of full SSL support in msb-java (including certificate validation).